doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -30,31 +30,44 @@ development environment with Docker may be more reliable.*
 
    .. note::
 
       If ``conda`` is not a recognized command, try restarting your terminal. If it is still not recognized, please see "Should I add Anaconda to the macOS or Linux PATH?" in the `Anaconda FAQ`_.
 
-#. Enter ``conda create --name scipydev``. |br| This tells ``conda`` to create a virtual environment named ``scipydev``. Note that ``scipydev`` can be replaced with any name you'd like to refer to your virtual environment.
+#. Enter ``conda create --name scipydev python=3.8 numpy pybind11 cython pytest gfortran_linux-64 gxx_linux-64 sphinx matplotlib mypy git``. |br| This tells ``conda`` to create a virtual environment named ``scipydev`` (or another name that you prefer) with several packages.
 
-#. You're still in the base environment. Activate your new virtual environment by entering ``conda activate scipydev``. |br| If you're working with an old version of ``conda``, you might need to type ``source activate scipydev`` instead (see `here <https://stackoverflow.com/questions/49600611/python-anaconda-should-i-use-conda-activate-or-source-activate-in-linux)>`__. Note that you'll need to have this virtual environment active whenever you want to work with the development version of SciPy.
+   * ``numpy pybind11 cython pytest`` are four packages that SciPy depends on.
+
+   * ``gfortran_linux-64 gxx_linux-64`` are compilers used to build SciPy's Fortran, C, and C++ source code.
+
+   * ``sphinx`` and ``matplotlib`` are required to render the SciPy documentation.
+
+   * ``mypy`` is a static type checker for Python. Consider using it.
+
+   * ``git`` is a version control system used to download and manage the SciPy source code.
 
-#. (Optional) Enter ``conda list`` again. Note that the new virtual environment has no packages installed. If you were to open a Python interpreter now, you wouldn't be able to import ``numpy``, ``scipy``, etc...
+   Note that we're installing SciPy's build dependencies and some other software, but not SciPy itself.
 
-#. Enter ``conda install cython numpy pytest spyder pybind11``. |br| Note that we're only installing SciPy's build dependencies (and Spyder so we can use the IDE), but not SciPy itself.
+.. note::
 
-#. `Rename the file`_ ``anaconda3/envs/scipydev/lib/libgfortran.so`` to ``anaconda3/envs/scipydev/lib/libgfortran.so_backup``, where ``anaconda3`` is to be replaced with the full path of your Anaconda installation. *Note: This file provides an incorrect Fortran shared library; renaming it forces the system to find the right one for SciPy. Other libraries, however, might rely on this version, so we suggest that you only use this environment for SciPy development.*
+	You could ``conda create`` an empty virtual environment first, then
+	``conda install`` the packages, but creating the virtual environment
+	with all the packages you need is preferable to installing packages
+	individually because it makes it easier for ``conda`` to solve
+	the package dependencies optimally.
```

Comment:
```suggestion
      You could ``conda create`` an empty virtual environment first, then
      ``conda install`` the packages, but creating the virtual environment
      with all the packages you need is preferable to installing packages
      individually because it makes it easier for ``conda`` to solve
      the package dependencies optimally.
```